Libplanet/Blockchain/KeyConverters.cs

```diff
@@ -12,5 +12,8 @@ internal static class KeyConverters
 
         internal static string ToFungibleAssetKey((Address, Currency) pair) =>
             ToFungibleAssetKey(pair.Item1, pair.Item2);
+
+        internal static string ToTotalSupplyKey(Currency currency) =>
+            "__" + ByteUtil.Hex(currency.Hash.ByteArray).ToLowerInvariant();
```

Comment:
I'm sure there is a reason, but just out of curiosity, what's with the magic string `"__"`? 😶

Reply:
It's out of thought that if the key for FAV is `_(address)_(currency)`, having the total supply of a currency (which might be as well as thought of the "global" balance) might as well as be `_(nothing)_currency`. I've also thought about having something like `_system_(currency)`, but after @dahlia and I had a conversation about whether we need a system namespace for state, we thought that it might be unnecessary.